string robotName
int32[] amount
